tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -21,7 +21,7 @@
 
 @pytest.mark.regions(["us-east-1"])
 @pytest.mark.instances(["c5.xlarge"])
-@pytest.mark.oss(["centos7", "alinux"])
+@pytest.mark.oss(["centos7", "alinux", "ubuntu1604", "ubuntu1804"])
```

Comment:
what about centos6?

Reply:
We're not supporting Lustre on Centos 6. See https://docs.aws.amazon.com/fsx/latest/LustreGuide/install-lustre-client.html